docs/contributing/pull-requests/index.md

```diff
@@ -114,8 +105,13 @@ Please follow the following checklist when creating a PR.
     (e.g. EC-123) or a link to the ticket anywhere in the PR title or body.
 - Add appropriate reviewer(s).
   - In most cases, this should be your pod. Each pod should have a GitHub team to simplify
-    requesting reviews, for example the _Platform pod_ has `pod-ps-dev`. There is a wider group
-    which includes QA and designers, `pod-ps`.
+    requesting reviews. The pod teams are named as follows:
+    - `@pod-{podAbbreviation}`: This will notify the entire pod, including product, design, QA and
```

Comment:
Do we want to look into adding a group for the design people in each pod? Or keep it as direct references by the pod?

Reply:
I've reached out to Danielle to get her opinion on this.

Reply:
They already have the `dept-design` tag, so I added that to the docs.